test/acceptance/cli-test.js

```diff
@@ -131,9 +131,14 @@ describe('ember-template-lint executable', function () {
     });
 
     describe('given no path', function () {
-      it('should print errors', function () {
+      // you don't have to specify path on Windows OS
+      if (process.platform === 'win32') {
```

Comment:
I wonder if it is not a bug 😅. Could you push a commit without this guard clause to highlight it in CI?

Reply:
I've added the check. Should pop up during the CI build...

Reply:
Yep, it's there. See the CI logs. Instead of offering me to provde the correct arguments it shows that it actually does the linting and gives output.